src/coreclr/src/pal/src/synchobj/semaphore.cpp

```diff
@@ -54,87 +54,43 @@ CAllowedObjectTypes aotSempahore(otiSemaphore);
 
 /*++
 Function:
-CreateSemaphoreExA
-
-Note:
-lpSemaphoreAttributes currently ignored:
--- Win32 object security not supported
--- handles to semaphore objects are not inheritable
-
-Parameters:
-See MSDN doc.
---*/
-
-HANDLE
-PALAPI
-CreateSemaphoreExA(
-        IN LPSECURITY_ATTRIBUTES lpSemaphoreAttributes,
-        IN LONG lInitialCount,
-        IN LONG lMaximumCount,
-        IN LPCSTR lpName,
-        IN /*_Reserved_*/  DWORD dwFlags,
-        IN DWORD dwDesiredAccess)
-{
-    // dwFlags is reserved and unused, and dwDesiredAccess is currently
-    // only ever used as SEMAPHORE_ALL_ACCESS.  The other parameters
-    // all map to CreateSemaphoreA.
-    _ASSERTE(SEMAPHORE_ALL_ACCESS == dwDesiredAccess);
-
-    return CreateSemaphoreA(
-        lpSemaphoreAttributes,
-        lInitialCount,
-        lMaximumCount,
-        lpName);
-}
-
-/*++
-Function:
-  CreateSemaphoreA
+  CreateSemaphoreW
 
 Note:
-  lpSemaphoreAttributes currently ignored:
+  lpSemaphoreAttributes currentely ignored:
```

Comment:
```suggestion
  lpSemaphoreAttributes currently ignored:
```

Reply:
Looks like same typo was present in multiple places under `synchobj` directory; fixed all the occurrences.